server/bin/pbench-backup-tarballs.py

```diff
@@ -48,28 +48,10 @@ def __init__(
 
 
 def sanity_check(lb_obj, s3_obj, config, logger):
-    # make sure archive is present
-    archive = config.ARCHIVE
-    archivepath = os.path.realpath(archive)
-
-    if not archivepath:
-        logger.error(
-            "The ARCHIVE directory {}, does not resolve to a real location", archive
-        )
-        return None, None
-
-    if not os.path.isdir(archivepath):
-        logger.error(
-            "The ARCHIVE directory {}, does not resolve {} to a directory",
-            archive,
-            archivepath,
-        )
-        return None, None
-
     # make sure the local backup directory is present
     backup = config.BACKUP
 
-    if len(backup) == 0:
+    if not backup:
```

Comment:
This should be an if./else: if backup is not defined then set lb_obj to None, but don't do the rest of that stuff that depends on a valid backup. So the try/except should be inside an else.